cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -32,41 +32,45 @@
 {% set page_url = get_protected_url(ancestor) %}
 
 <div class="o-item-introduction">
-    <div class="h4">
+    {% if value.category is string %}
         {{ category_slug.render(value.category, page_url, index=form_id) }}
-    </div>
-    <h1>{{ value.heading }}</h1>
+    {% else %}
+        {% for category in page.categories.all() %}
+            {{ '|' if loop.index > 1 else '' }}
+            {{ category_slug.render(category.name, get_protected_url(ancestor), 'post_slug', index=index) }}
+        {% endfor %}
+    {% endif %}
+    <h1>{{ value.heading | safe }}</h1>
 
     {% if value.paragraph %}
         <div class="lead-paragraph">{{ parse_links(value.paragraph) | safe }}</div>
     {% endif %}
+
     {% if value.date or page.authors.all() %}
-        <p>
+        <div class="meta">
     {% endif %}
-    {% for author in page.authors.names() %}
-        {% if loop.index == 1  %}
-            By <a href="{{ page_url }}?filter{{ form_id }}_authors={{ author }}">
-                {{ author }}
-                </a>
-        {% elif loop.last == true %}
-            and <a href="{{ page_url }}?filter{{ form_id }}_authors={{ author }}">
-                {{ author }}
-                </a>
-        {% else %}
-            , <a href="{{ page_url }}?filter{{ form_id }}_authors={{ author }}">
-                {{ author }}
+    {% set num_authors = page.authors.all()|length %}
+        {% if num_authors %}
+            <span class="byline">
+            {%- for author in page.authors.names() -%}
+                {{- 'By ' if loop.index == 1 else ' and ' }}
+                <a href="{{ page_url }}?filter{{ form_id }}_authors={{ author }}">
+                    {{ author }}
                 </a>
+            {%- endfor -%}
```

Comment:
Oh, I see...hmm. The authors are a list of names added as tags to the page, so I wouldn't think that would happen, unless there's the possibility of an empty tag?